### ISSUE TITLE
Determinen domain name for Emory production server

### ISSUE THREAD

**Issue by shanitajohn** [curationexperts/laevigata#90]

**Comment by fpici** [comment]
@Hi Shanita. I will defer this question to Bethany when she returns on Tuesday May 30th :-) @bnash108

**Comment by bnash108** [comment]
@shanitajohn We currently have the migration of old ETD data as possibly out of scope for Phase I of this project, which means we'll need to maintain access to the old site for older ETDs while we have the new site up. Unless we think the data migration will happen before we go live with the new site?

If we do need to have both sites available, @fpici and I will need to discuss with Lisa and Rosalyn on this end about best approach for domains.

**Comment by shanitajohn** [comment]
Asking @mark-dce to weigh in. I know we were hoping to do a migration test before go live.

**Comment by mark-dce** [comment]
We were assuming a complete data migration prior to the first new student deposits.  I think it might be somewhat confusing to have the old system still running unless Emory can clearly disable the submission process on it (and ideally set it up to redirect to the new system).  So the main options we should discuss pros and cons of are:

1) Complete migration from legacy system before accepting new submissions
2) Run systems in parallel
    a) under separate domains
    b) having legacy application redirect submitters to the new application

If the solution involves having two systems live in parallel, we will need to determine which domain remains after the second system is shut down and how the other domain is 'retired'.

**Comment by shanitajohn** [comment]
**ANSWER:** 
- Emory has decided to  use the current ETD domain name for the Laevigata application 
- Plan is to switch domains on August 12, 2017
